.cspell.json

```diff
@@ -493,6 +493,7 @@
     "WCAG",
     "stackedplot",
     "Andale",
+    "unnnormalized",
```

Comment:
I'm prepared to accept unnormalized at a stretch. But unnnormalized? With three n's?